src/pages/Staking/index.tsx

```diff
@@ -106,7 +107,15 @@ const Staking = (): JSX.Element => {
   const { t } = useTranslation();
   const prices = useGetPrices();
 
-  const { governanceTokenBalance, bridgeLoaded, address } = useSelector((state: StoreType) => state.general);
+  const { bridgeLoaded, address } = useSelector((state: StoreType) => state.general);
+
+  const {
+    isIdle: governanceTokenBalanceIdle,
```

Comment:
Hi @crypto-engineer, could we rename the return values in the hook so that we use the hook with 

```
const {
    governanceTokenBalanceIdle,
    governanceTokenBalanceIsLoading,
    governanceTokenBalance
} = useGovernanceTokenBalance();
```

As with the error handler, it would be nice to handle this once in the hook, rather than having to do in each component.

Reply:
Done!